admin/admin-domain-service/admin-service/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
+    id 'net.researchgate.release'
+}
```

Comment:
shall we remove this as release plugin is added to the root project?

Reply:
Fixed with [ca3ec70](https://github.com/wso2/apk/pull/138/commits/ca3ec702a70164c55f4f85fed7694b02286dc96c)